src/adapter_interface.js

```diff
@@ -0,0 +1,7 @@
+const assertParam = require('./utils/assert-param')
+
+module.exports.create = config => {
+    assertParam(config, 'provider')
+    let provider = config.provider
+    return require(`./adapters/${provider}.js`)
```

Comment:
Could you add semicolon at least at the end of the 'return' statements? Sometimes it could be helpful to avoid bugs with very complex return statements.

Reply:
Yes :) I appreciate your comments.